src/assignments/2-operator.js

```diff
@@ -14,6 +14,25 @@ output example
 */
 // start here writing the code for program1
 
+const num1 = Number(prompt("Number 1: "));
+const num2 = Number(prompt("Number 2: "));
+
+const add = num1 + num2;
+console.log(`${num1}+${num2}=${add}`);
+
+const sub = num1 - num2;
+console.log(`${num1}-${num2}=${sub}`);
+
+const multiply = num1 * num2;
+console.log(`${num1}*${num2}=${multiply}`);
+
+const divide = (num1 / num2).toFixed(2);
+console.log(`${num1}/${num2}=${divide}`);
+
+const remain = num1 % num2;
+console.log(`${num1}%${num2}=${remain}`);
```

Comment:
here is also 
 in this case , it is quite necessary to check whether the num2 > 0 or not, then do the calculation.

Reply:
Now I understand why thank you!